.requirements

```diff
@@ -21,7 +21,7 @@ LUA_RESTY_WEBSOCKET=966c69c39f03029b9b42ec0f8e55aaed7d6eebc0 # 0.4.0.1
 ATC_ROUTER=ffd11db657115769bf94f0c4f915f98300bc26b6 # 1.6.2
 SNAPPY=23b3286820105438c5dbb9bc22f1bb85c5812c8a # 1.2.0
 
-KONG_MANAGER=nightly
+KONG_MANAGER=v3.8.0.0
```

Comment:
Should the version be 3.8.0 for CE?

Reply:
No, `kong-manager` always uses 4-digit versions:

https://github.com/Kong/kong/blob/e89feb5b85286394330968054c02ac851fd2b781/.requirements#L18
https://github.com/Kong/kong/blob/4e006aa85d7de34668ad40d601e275b575f0d2aa/.requirements#L15